### DISCUSSION TITLE
Update Revios or Reviostool

### DISCUSSION THREAD

**Discussion by lovi-dzen** [meetrevision/playbook#64] [Q&A]
Good afternoon, how can I update Revios ? After installing (for example Revios 23.10) how can I install Revios 23.12 preview 5 ? Do I install it on top or do I need to install it on a clean system ? Fixes are made, but how to apply them to your system is not clear ..... help please

**Comment by melo936** [comment]
https://www.revi.cc/docs/playbook/install#update

**Reply by lovi-dzen** [reply]
"...If you want to update your Windows version..." as far as I understand it, it's a windows update. If I install the latest version of Revios on the previous one, my programmes will not be lost, what about the applications from the Microsoft shop ?

**Comment by vovanvova1** [comment]
Hello, when should we expect a new version of reviOS?
